sdk/database/helper/connutil/sql.go

```diff
@@ -43,6 +43,7 @@ type SQLConnectionProducer struct {
 	ServiceAccountJSON       string      `json:"service_account_json" mapstructure:"service_account_json" structs:"service_account_json"`
 	DisableEscaping          bool        `json:"disable_escaping" mapstructure:"disable_escaping" structs:"disable_escaping"`
 	usePrivateIP             bool        `json:"use_private_ip" mapstructure:"use_private_ip" structs:"use_private_ip"`
+	usePSC                   bool        `json:"use_psc" mapstructure:"use_psc" structs:"use_psc"`
```

Comment:
Are these fields returned in the READ /config endpoint?

Reply:
Same response as earlier.
`usePrivateIP` was already set like this.

Reply:
I have exported both field after my test.